src/browser/modules/Stream/SysInfoFrame/SysInfoFrame.tsx

```diff
@@ -81,20 +83,64 @@ export class SysInfoFrame extends Component<
     idAllocation: [],
     pageCache: [],
     transactions: [],
-    error: '',
+    errorMessage: null,
     results: false,
-    success: false,
     autoRefresh: false,
-    autoRefreshInterval: 20 // seconds
+    autoRefreshInterval: 20, // seconds
+    namespacesEnabled: false,
+    userConfiguredPrefix: 'neo4j'
   }
-  helpers = this.props.hasMultiDbSupport ? helpers : legacyHelpers
 
   componentDidMount(): void {
-    this.getSysInfo()
+    this.getSettings()
```

Comment:
It could cause problems that we only fetch the settings once on mount, but should work fine for the most common cases

Reply:
Opted to to do this to spam the server less, detecting server changes is an area of weakness for sure.